15.-Project-3-Elections-Scraper.py

```diff
@@ -0,0 +1,155 @@
+
+import requests
+from bs4 import BeautifulSoup as BS
+import csv
+import sys
+import os
+import unicodedata
+
+URL = 'https://volby.cz/pls/ps2017nss/ps3?xjazyk=CZ'
+payload = {'area':None,
+           'municipality': None}
+
+
+def parse_command_line():
+    while True:
+        if len(sys.argv) != 3:
+            print('Not correct number of arguments (2)')
+            break
+        else:
+            arg = sys.argv[1:]
+            count = 0
+            for k, v in payload.items():
+                payload[k] = arg[count]
+                count += 1
+            return payload
```

Comment:
Isn't
```python
payload = {}
paylod['area'] = sys.argv[1]
payload['municipality'] = sys.argv[2]
```
more readable? 
If you want to make this better use library https://docs.python.org/3.8/library/argparse.html

Reply:
When you have command-line argument you should write some info about how to use it. What arguments it expects and what it does. Either write it to documentation or you can write a usage comment at the top of the file. Right now I don't know how to use the script.